metadata/src/test/java/org/apache/kafka/controller/ReplicationControlManagerTest.java

```diff
@@ -208,6 +212,24 @@ void unfenceBrokers(Integer... brokerIds) throws Exception {
             }
         }
 
+        void fenceBrokers(Set<Integer> brokerIds) throws Exception {
+            time.sleep(BROKER_SESSION_TIMEOUT_MS);
```

Comment:
nit: we can assert the `fencedBrokerIds` is empty before sleep to timeout session.

Reply:
Good call.. Although I would not do it in the common utility function. 
Will do it in the test method where this should hold.